kodi_voice/kodi.py

```diff
@@ -50,6 +50,14 @@ def sanitize_name(media_name, remove_between=False, normalize=True):
   name = name.strip()
   return name
 
+def sanitize_channel(name, lang='en'):
+  name = name.lower()
+  name = name.replace("+", " plus ")
+  name = re.sub(r"\sch\s", " channel ", name)
+  name = re.sub("^channel", "", name)
+  name = re.sub(r"(?<=\D)(?=\d)|(?<=\d)(?=\D)", " ", name)
+  name = words2digits(name, lang=lang)
```

Comment:
With the exception of removing `channel` I'd say yes, that's fine.  But if Kodi adds `channel` everywhere, they probably should be in the slot items too.